presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestPasswordAuthentication.java

```diff
@@ -79,9 +82,14 @@ public TestPasswordAuthentication()
     public final void destroy()
             throws IOException
     {
-        assertions.close();
-        elasticsearch.stop();
-        client.close();
+        Closeables.closeAll(
+                () -> assertions.close(),
```

Comment:
method reference?

Reply:
no, because i want the `closeAll` to be invoked even if one of the references is null